rocAL/rocAL_pybind/example/coco_pipeline.py

```diff
@@ -9,14 +9,12 @@
 from amd.rali.pipeline import Pipeline
 import amd.rali.ops as ops
 import amd.rali.types as types
-
-
 import sys
 import numpy as np
 
 
 class COCOPipeline(Pipeline):
-    def __init__(self, batch_size, num_threads, device_id, seed, data_dir, ann_dir, default_boxes,  crop, rali_cpu=True):
```

Comment:
@swetha097 can you look at this line - https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=7580035